### ISSUE TITLE
Apache httpd auto-instrumentation clone doesn't use configPath

### ISSUE THREAD

**Issue by pl4nty** [open-telemetry/opentelemetry-operator#1821]
The initContainer clone doesn't seem to respect the Instrumentation configPath argument, resulting in:

```
cp: cannot stat '/usr/local/apache2/conf/*': No such file or directory
```

https://github.com/open-telemetry/opentelemetry-operator/blob/846046ccccf3d2bde63be9cb7f75537af5332638/pkg/instrumentation/apachehttpd.go#L86-L87

**Comment by chrlic** [comment]
pls assign to me

**Comment by pl4nty** [comment]
Fixed in 0bfbea9e250899365856173d842bee4b797bc42b
